### DISCUSSION TITLE
compilation failed

### DISCUSSION THREAD

**Discussion by jenisha-changani** [Uniswap/interface#2500] [Q&A]
./src/constants/locales.ts
Module not found: Can't resolve '../locales/en-US' in 'C:\jenisha\Reactjs\practice_projects\interface\src\constants'

**Comment by BDMChau** [comment]
Any updates? I also stuck

**Reply by dydent** [reply]
Have you tried to update to the latest nodejs version and then run yarn install again? :D

**Reply by jenisha-changani** [reply]
> Any updates? I also stuck

I have found a way. All you have to do is just run the following commands in your terminal.

npm install touch-cli -g
yarn prei18n:extract
yarn i18n:extract
yarn i18n:compile
yarn graphql:generate

**Reply by Deacon77** [reply]
And the commands will ???

**Reply by Deacon77** [reply]
Ps- I'm just a beginner!
